data/human/war reactions.txt

```diff
@@ -0,0 +1,248 @@
+					goto scowl
+			label nobody
+			`	Both of them give hard belly laughs, sending waves through their mustaches from the breeze of their breath.`
+			`	The left twin says, "Who are you, Admiral Danforts?" His twin corrects him. "Danforth. Admiral Danforth. Did the Navy fire you or are you here on holiday?"`
+			choice
+				`	"Have you been under a rock the whole war? I'm Captain <first> <last>!"`
+				`	"Never mind."`
+					goto shrug
+				`	"At least I don't have a disheveled hamster stuck to my upper lip."`
+					goto scowl
+			`	Both twins look confused, and the left one responds. "We've been mining asteroids the whole war. The war was a big deal, but most of us had to keep living our lives. Do you think we have time to keep track of every warring captain?"`
+			`	The right one says apologetically, "Thank you for your service, Captain <last>."`
+			`	They both give clumsy but well-intentioned salutes and return to their ships.`
+				decline
+			label eyes
+			`	The left twin laughs and says, "Pardon us if we keep our eyes open and trust to our own guns." Both twins nod and depart to their ships.`
+				decline
+			label shrug
+			`	Both twins shrug and leave for their ships.`
+				decline
+			label scowl
+			`	Both twins' faces twist in a way that might indicate embarrassment or annoyance. It's hard to tell since their facial hair is halting any emotive power. They walk away in the direction of their ships.`
+				decline
+
+
+mission "War Reactions: Navy and Free Worlds Captains"
+	landing
+	minor
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		attributes "spaceport"
+		# This distance from Sirius puts them a few jumps from the edge of human space, or anywhere in the Free Worlds
+		near "Sirius" 5 100
+	to offer
+		has "event: post-war reactions begin"
+		random < 30
+	on offer
+		set "pug media coverage"
+		conversation
+			`You are relaxing in a cafe while your ship refuels, and a pair of middle-aged captains come to your table. One is Navy, the other is Free Worlds, and they're obviously sisters.`
+			`	"Captain Rain, R.N.S. Camuy," says one. "How's the <ship>, Captain <last>?"`
+			`	"Captain Fiona, F.S. Baobab," says her sister. "We both saw you in battle. You're the war's most accomplished mercenary!`
+			`	"We've been patrolling together and we saw your ship, so we've come to say 'hi.'"`
+			label questions
+			choice
+				`	"Why are Navy and Free Worlds captains traveling together?"`
+					goto navy
+					to display
+						not "War Reactions: Navy and Free Worlds Captains: label navy"
+				`	"Have you found any new aliens?"`
+					goto aliens
+					to display
+						not "War Reactions: Navy and Free Worlds Captains: label aliens"
+						has "War Reactions: Navy and Free Worlds Captains: label navy"
+				`	"Did you fight on the other side of the war from your sister?"`
+					goto sister
+					to display
+						not "War Reactions: Navy and Free Worlds Captains: label sister"
+				`	"Mercenary? I'm not a mercenary."`
```

Comment:
```suggestion
						not "War Reactions: Navy and Free Worlds Captains: label sister"
				`	"You were on opposite sides as Sisters!?"`
					goto sister2
					to display
						has "War Reactions: Navy and Free Worlds Captains: label sister"
						not "War Reactions: Navy and Free Worlds Captains: label sister2"
				`	"Mercenary? I'm not a mercenary."`
```

Reply:
Should 'Sisters' be capitalised?

Reply:
Indeed, not, the copypasta monster came and did it. I've fixed it.